app/views/content/funding-and-support/if-youre-a-veteran.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Funding and support if you're a veteran"
+heading: "Support training to teach if you're a veteran"
+description: |-
+    Find out how to get support when training to teach if you're a veteran.
+related_content:
+    Salaries and benefits of teaching: "/salaries-and-benefits"
+    Teach in further education without a degree: "https://www.teach-in-further-education.campaign.gov.uk/"
+    David’s journey from the army to teacher training: "/blog/from-the-army-to-teacher-training"
+promo_content:
+    - content/funding-and-support/promos/get-adviser-veterans-promo
+navigation: 20.30
+navigation_title: If you're a veteran
+navigation_description: Find out how to get support when training to teach if you're a veteran.
+calls_to_action:
+  chat:
+    name: chat
+    arguments: {}
+keywords:
+    - veterans
+    - service leavers
+    - army
+    - military
+---
+
+If you’re transitioning from the armed forces or your service has already ended, you could be eligible for support to train to teach.
+
+As a veteran, you can bring a unique perspective to teaching by using your organisation, leadership and management skills in the classroom.
+
+There are full and part-time teacher training courses available.
+
+## Undergraduate bursary for veterans
+
+You may be eligible for a tax-free undergraduate bursary of £40,000 if you’re:
+
+* a veteran
+* training to teach secondary biology, physics, chemistry, computing, maths or languages
+* doing an undergraduate degree leading to [qualified teacher status (QTS)](/what-is-qts) in England
+
+Doing an undergraduate degree leading to QTS means that you would be qualified to teach in England.
+
+You do not need to apply for a bursary. Your teacher training provider will determine whether you are eligible and will then pay you in monthly instalments across the final 2 years of your course.
```

Comment:
@EllieNodder is this sentence saying you don't need to apply for a bursary to get one? Or you don't need to apply for a bursary to train to teach?